library/extractor/src/main/java/com/google/android/exoplayer2/extractor/mp4/PsshAtomUtil.java

```diff
@@ -177,9 +189,17 @@ private static PsshAtom parsePsshAtom(byte[] atom) {
       return null;
     }
     UUID uuid = new UUID(atomData.readLong(), atomData.readLong());
+    UUID[] keyIds = null;
     if (atomVersion == 1) {
       int keyIdCount = atomData.readUnsignedIntToInt();
-      atomData.skipBytes(16 * keyIdCount);
+//      atomData.skipBytes(16 * keyIdCount);
```

Comment:
Looks like we are changing some of the source code of ExoPlayer here, is this what we need? 

I thought we were just adding logs